src/routes/auth.js

```diff
@@ -46,12 +46,45 @@ router.post('/signup', async (request, response, next) => {
       request,
       user: newUser,
     });
+
+    const confirmToken = jwt.sign({ email: newUser.email }, '1y');
+
+    const confirmLink = `${config.CLIENT_URL}/confirm-email?token=${confirmToken}`;
+
+    await emailService.sendHtmlEmail({
+      to: newUser.email,
+      subject: 'Confirm your Email',
+      html: `<p>Hello ${newUser.name},<br/><br/>This is an automated message sent to you to confirm your email. Please click this <a href=${confirmLink}>link</a> confirm your email</p>`,
+    });
     response.json({ accessToken, refreshToken });
   } catch (error) {
     next(error);
   }
 });
 
```

Comment:
Do these changes work? Have you tried yarn test?

Reply:
Yes, I've tried these on yarn tests. They weren't working because it was missing parameters for sending the HTML email. I will fix it by using a mock value through jest.